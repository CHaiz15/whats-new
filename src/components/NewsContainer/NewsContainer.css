.NewsContainer {
  display: flex;
  flex-flow: wrap;
  margin: 0;
}
