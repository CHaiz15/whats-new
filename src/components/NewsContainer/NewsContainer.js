import React from 'react';
import './NewsContainer.css'
import NewsArticle from '../NewsArticle/NewsArticle.js'

const NewsContainer = () => {
  
  return <div> </div>;
}

export default NewsContainer;
