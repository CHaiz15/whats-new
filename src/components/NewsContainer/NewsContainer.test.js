import React from 'react'
import { shallow } from 'enzyme'
import NewsContainer from './NewsContainer.js'

describe('NewsContainer', () => {
  it('renders an indication of no newsSource', () => {
    const wrapper = shallow(<NewsContainer
      newsSource={[]}
    />);

    expect(wrapper).toMatchSnapshot();
  })
  it('renders an indication of newsSource', () => {
    const wrapper = shallow(<NewsContainer
      newsSource={[{headline:'Broncos lose', img:'bronco.png', url:'gobroncos.com', description:'Broncos miss the playoffs'}]}
    />);

    expect(wrapper).toMatchSnapshot();
  })
})
