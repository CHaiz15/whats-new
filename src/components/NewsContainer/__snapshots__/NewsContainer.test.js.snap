// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContainer renders an indication of newsSource 1`] = `
<div
  className="NewsContainer"
>
  <NewsArticle
    description="Broncos miss the playoffs"
    headline="Broncos lose"
    img="bronco.png"
    key="Broncos lose"
    url="gobroncos.com"
  />
</div>
`;

exports[`NewsContainer renders an indication of no newsSource 1`] = `
<div
  className="NewsContainer"
/>
`;
